### PR TITLE
LPD-6913 | master

### PR DESCRIPTION
**Pull request by rachaelkoestartyo**

### CODE REVIEW

**Comment by rachaelkoestartyo** [comment]
ci:test:sf

**Comment by rachaelkoestartyo** [comment]
ci:test:relevant

**Comment by rachaelkoestartyo** [comment]
ci:test:sf

**Comment by rachaelkoestartyo** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/osb-faro-service/src/main/java/com/liferay/osb/faro/service/impl/FaroProjectLocalServiceImpl.java [hunk 1]
```diff
@@ -322,6 +323,10 @@ public FaroProject updateSubscription(
 			return null;
 		}
 
+		if (faroProject.getSubscription() != subscription) {
```
Comment (marcosapmf) [inline modules/dxp/apps/osb/osb-faro/osb-faro-service/src/main/java/com/liferay/osb/faro/service/impl/FaroProjectLocalServiceImpl.java:326]: @rachaelkoestartyo, I think we should use `subscription.get('name')` instead. `subscription` is a JSON that stores some information that are frequently updated, like `individualsCount` and `pageViewsCount`.

**Review by marcosapmf — CHANGES_REQUESTED** [review]

**Review by marcosapmf — APPROVED** [review]

**Comment by marcosapmf** [comment]
ci:forward

**Comment by marcosapmf** [comment]
ci:forward:force

**Pull request closed without merging**